### PR TITLE
Add percentages to results

### PR DESCRIPTION
**Pull request by mmathesius**
Add percentage column to the build status overview table.

Also write just the success rate percentage to a file so it can be included on a dashboard page.

(Added at the request of Rui.)

### CODE REVIEW

**Comment by tdawson** [comment]
Looks good.  Thanks for this.
I'm holding off merging until we've got jenkins working again.  I'd like one successful run of periodic before mergin, so we aren't wondering if this is the problem or not.

File: eln-check.py [hunk 1]
```diff
@@ -297,13 +301,29 @@ def diff_with_rawhide(package, eln_build=None, rawhide_build=None):
...
+      percent_old = percentage_old,
       count_none = counter_none,
-      count_total = counter_same + counter_old + counter_none,
+      percent_none = percentage_none,
+      count_total = counter_total,
       packages = package_list
       ))
     w.close()
+
+    r = open(args.successrate,'w')
```
Comment (sturivny) [inline eln-check.py:327]: The best way to close a file is by using the `with` statement. This ensures that the file is closed when the block inside the with statement is exited.

We don't need to explicitly call the close() method. It is done internally.
https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files

**Review by sturivny — CHANGES_REQUESTED** [review]
Please, use `with` statement for working with files

File: eln-check.py (hunk 1 above)
Comment (tdawson) [reply eln-check.py:327]: I'm not sure I understand this comment saying that he needs to change his pull request.
He was following the convention of the rest of the python file.
If you feel it is best to use the with statement, don't hijack this pull request.  Please create a new one that fixes all of them.

**Comment by tdawson** [comment]
I did find one problem with this.  It's that the file will not be shown in Jenkins.
You need to edit the Jenkins file in this repo. At the top add the filename as a variable (this part is optional, but good) and at the bottom in the archiveArtifacts line, add that variable.  Anything that isn't in the archiveArtifacts, doesn't show up in the area.

File: eln-check.py (hunk 1 above)
Comment (mmathesius) [reply eln-check.py:327]: @sturivny Thank you for the feedback. I am aware of the convention using the `with` statement and I _almost_ implemented it that way. However, I felt it was better practice to stick with the existing style of the script.

**Comment by mmathesius** [comment]
> I did find one problem with this. It's that the file will not be shown in Jenkins.
> You need to edit the Jenkins file in this repo. At the top add the filename as a variable (this part is optional, but good) and at the bottom in the archiveArtifacts line, add that variable. Anything that isn't in the archiveArtifacts, doesn't show up in the area.

@tdawson Thanks for catching that! I added a commit to take care of `Jenkinsfile`.

**Pull request merged**